src/mcts/search.cc

```diff
@@ -578,7 +581,7 @@ SearchWorker::NodeToProcess SearchWorker::PickNodeToExtend() {
         // To ensure we have at least one node to expand, always include
         // current best node.
         if (child != search_->best_move_node_ &&
-            search_->remaining_playouts_ <
+            p_early_exit*search_->remaining_playouts_ <
```

Comment:
Where is this variable defined?
Also it's one additional float multiplication per visit. Probably not noticeable at all, but would be interesting to  check with `--backend=random`